src/mmap.h

```diff
@@ -138,12 +138,15 @@ class MemoryMap
 	// Stays in the I/O Ports at 0xFF4B
 	Byte* reg_WX;
 
+	// Audio Write Listener
+	// updates and writes back after audio write
+	std::function<void(Word)> audioWriteHandler;
+
 public:
 	// Audio Unit
```

Comment:
remove these comments

Reply:
These comments are for description. Do you really want them gone?

Reply:
APU pointer is not longer in memory, but comments explaining them are still there.
```
// Audio Unit
```

Reply:
ah. that one!